editor/src/components/editor/persistence/generic/persistence-machine.ts

```diff
@@ -743,6 +748,7 @@ export function createPersistenceMachine<ModelType, FileType>(
                         },
                       }),
                     },
+                    // FIXME Error handling
```

Comment:
Is this what I saw you (elsewhere) comment on that some further fix was needed?

Reply:
Yeah, I have a follow up PR I'll be opening in a sec that deals with this